MCprep_addon/util.py

```diff
@@ -228,7 +240,11 @@ def min_bv(version: Tuple, *, inclusive: bool = True) -> bool:
 
 
 def bv28() -> bool:
-	"""Check if blender 2.8, for layouts, UI, and properties. """
+	"""
+	Check if blender 2.8, for layouts, UI, and properties. 
+	
+	Deprecated in MCprep 3.5, but kept to avoid breakage for now...
+	"""
 	env.deprecation_warning()
 	return min_bv((2, 80))
```

Comment:
I believe we have already removed all references to this, haven't we? I'd be in favor of just removing now.

Reply:
Resolved in 1fec0da624d75f615590469ef1ca34b30ad432c5